src/sentry/monitors/migrations/0001_initial.py

```diff
@@ -0,0 +1,387 @@
+# Generated by Django 5.1.7 on 2025-04-03 17:01
+
+import django.db.models.deletion
+import django.utils.timezone
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+import sentry.db.models.fields.hybrid_cloud_foreign_key
+import sentry.db.models.fields.jsonfield
+import sentry.db.models.fields.slug
+import sentry.db.models.fields.uuid
+import sentry.monitors.models
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
```

Comment:
If we ever squash the migrations in `sentry` we'll need to set this up to have the correct initial state.

Reply:
Squashing migrations is always a tedious and time-consuming process. It might be easier to start future squash efforts in django apps.